tests/app/v2/notifications/test_post_notifications.py

```diff
@@ -34,6 +38,14 @@
     create_service_with_inbound_number,
     create_template,
 )
+from tests.conftest import set_config
+
+
+def rows_to_csv(rows):
+    output = StringIO()
+    writer = csv.writer(output)
+    writer.writerows(rows)
+    return output.getvalue()
```

Comment:
My preference is would be to have less logic in the tests and define CSVs in plain text like:
```
col_1,col_2\n
val_a,val_b\n
val_c,val_d\n
```
Since that makes the tests a bit easier to read (although longer). 
No need to change it but figured I would comment.